File: javaexample/src/main/java/io/cthonic/mythos/javaexample/ui/presenters/FusPresenter.java
package io.cthonic.mythos.javaexample.ui.presenters;

import android.support.annotation.NonNull;

import io.cthonic.mythos.mvp.PresenterImpl;
import io.cthonic.mythos.javaexample.ui.vus.FusVu;

/**
 * Created by jhavatar on 3/3/2016.
 */
public class FusPresenter extends PresenterImpl<FusVu> {

    public void attachVu(@NonNull FusVu vu) {
        super.attachVu(vu);

        vu.setTitle("FUS");
        vu.setBg("#FC6C85");
    }
}

File: kotlinexample/src/main/kotlin/io/cthonic/mythos/kotlinexample/ui/presenters/FusPresenter.kt
package io.cthonic.mythos.kotlinexample.ui.presenters

import io.cthonic.mythos.kotlinexample.ui.vus.FusVu
import io.cthonic.mythos.mvp.PresenterImpl

/**
 * Created by jhavatar on 3/3/2016.
 */
class FusPresenter() : PresenterImpl<FusVu>() {

    override fun attachVu(vu: FusVu) {
        super.attachVu(vu);

        vu.setTitle("FUS");
        vu.setBg("#FC6C85")
    }
}